quartz/components/scripts/search.inline.ts

```diff
@@ -110,21 +133,67 @@ document.addEventListener("nav", async (e: unknown) => {
     }
   }
 
+  function trimContent(content: string) {
```

Comment:
take a look at `quartz/plugin/transformers/description.ts`, maybe can reuse some logic here

Reply:
done in 19dbc357bd1d495353c8e07d7577edc71c68e4b8.

would be cool if we could trim the autolink headings from `gfm`, but i dont think we can access the `gfm` config in search?

Right now, the preview would look something like `First heading § next line starts here...`